benchmark/kvbench/runtime/etcd_rt.py

```
@@ -76,16 +84,37 @@ def __init__(
-        logger.info("ETCD client initialized with host %s & port %d", host, port)
+        logger.info(
+            "ETCD client initialized with host %s & port %d, rank=%d, world_size=%d",
+            host,
+            port,
+            self.rank,
+            self.world_size,
+        )
 
         try:
             self.client = etcd3.client(host=host, port=port)
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)